sections/announcement-bar.liquid

```diff
@@ -115,6 +130,87 @@
         </div>
       </slideshow-component>
     {%- endif -%}
+    <div class="desktop-localization-wrapper">
```

Comment:
I don't really like this classname because it's not itself a desktop-specific element. I think it might make more sense to use the small-hide type classes up here instead. That said, I think this is just a copy of existing markup from header.liquid, right? In which case I'd rather the approaches be consistent. I'm starting to feel like the locale code could benefit from a shared snippet.